Sources/App/commands.swift

```diff
@@ -0,0 +1,30 @@
+import Foundation
+import StevensonCore
+
+let fastlane = SlackCommand(
```

Comment:
`private`?

Reply:
it's being used in `configure.swift`, would like to keep things separate